src/shared/sleep-config.c

```diff
@@ -458,3 +474,13 @@ void free_sleep_config(SleepConfig *sc) {
 
         free(sc);
 }
+
+static SwapEntry* swap_entry_free(SwapEntry *se) {
+        if (se) {
+                free(se->device);
+                free(se->type);
+                free(se);
+        }
+
+        return NULL;
+}
```

Comment:
Just move this definition to the top, there's no need for a forward declaration.